src/PantheonCalculator.vala

```diff
@@ -45,7 +45,11 @@ namespace PantheonCalculator {
                     window.destroy ();
                 }
             });
+            SimpleAction undo_action = new SimpleAction ("undo", null);
+            undo_action.activate.connect (() => { window.undo ();} );
```

Comment:
Can we change the whitespace in this line to look like this?
`undo_action.activate.connect (() => { window.undo (); });`